### PR TITLE
Docker課題

### PR DESCRIPTION
**Pull request by yuta-yamaura**

### CODE REVIEW

**Comment by yuta-yamaura** [comment]
ターミナルからdocker-compose upしたログを添付致します。

user@usernoMBP rails-docker % docker-compose up
[+] Building 0.0s (0/0)                                                                                                                                                                docker:desktop-linux
[+] Running 2/2
 ✔ Container rails-docker-db-1   Created                                                                                                                                                               0.0s 
 ✔ Container rails-docker-web-1  Created                                                                                                                                                               0.0s 
Attaching to rails-docker-db-1, rails-docker-web-1
rails-docker-db-1   | 
rails-docker-db-1   | PostgreSQL Database directory appears to contain a database; Skipping initialization
rails-docker-db-1   | 
rails-docker-db-1   | 2024-02-18 08:45:54.761 UTC [1] LOG:  starting PostgreSQL 12.17 (Debian 12.17-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
rails-docker-db-1   | 2024-02-18 08:45:54.762 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
rails-docker-db-1   | 2024-02-18 08:45:54.762 UTC [1] LOG:  listening on IPv6 address "::", port 5432
rails-docker-db-1   | 2024-02-18 08:45:54.764 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rails-docker-db-1   | 2024-02-18 08:45:54.815 UTC [28] LOG:  database system was shut down at 2024-02-08 21:40:17 UTC
rails-docker-db-1   | 2024-02-18 08:45:54.835 UTC [1] LOG:  database system is ready to accept connections
rails-docker-db-1   | 2024-02-18 08:45:58.033 UTC [35] ERROR:  database "myapp_development" already exists
rails-docker-db-1   | 2024-02-18 08:45:58.033 UTC [35] STATEMENT:  CREATE DATABASE "myapp_development" ENCODING = 'unicode'
rails-docker-web-1  | Database 'myapp_development' already exists
rails-docker-db-1   | 2024-02-18 08:45:58.051 UTC [36] ERROR:  database "myapp_test" already exists
rails-docker-db-1   | 2024-02-18 08:45:58.051 UTC [36] STATEMENT:  CREATE DATABASE "myapp_test" ENCODING = 'unicode'
rails-docker-web-1  | Database 'myapp_test' already exists
rails-docker-web-1  | => Booting Puma
rails-docker-web-1  | => Rails 7.0.6 application starting in development 
rails-docker-web-1  | => Run `bin/rails server --help` for more startup options
rails-docker-web-1  | Puma starting in single mode...
rails-docker-web-1  | * Puma version: 5.6.6 (ruby 3.2.2-p53) ("Birdie's Version")
rails-docker-web-1  | *  Min threads: 5
rails-docker-web-1  | *  Max threads: 5
rails-docker-web-1  | *  Environment: development
rails-docker-web-1  | *          PID: 1
rails-docker-web-1  | * Listening on http://0.0.0.0:3000
rails-docker-web-1  | Use Ctrl-C to stop
rails-docker-web-1  | Started GET "/" for 192.168.65.1 at 2024-02-18 08:46:24 +0000
rails-docker-web-1  | Cannot render console from 192.168.65.1! Allowed networks: 127.0.0.0/127.255.255.255, ::1
rails-docker-web-1  |   ActiveRecord::SchemaMigration Pluck (1.0ms)  SELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC
rails-docker-web-1  | Processing by TasksController#index as HTML
rails-docker-web-1  |   Rendering layout layouts/application.html.erb
rails-docker-web-1  |   Rendering tasks/index.html.erb within layouts/application
rails-docker-web-1  |   Task Load (2.0ms)  SELECT "tasks".* FROM "tasks"
rails-docker-web-1  |   ↳ app/views/tasks/index.html.erb:6
rails-docker-web-1  |   Rendered tasks/_task.html.erb (Duration: 1.4ms | Allocations: 227)
rails-docker-web-1  |   Rendered tasks/index.html.erb within layouts/application (Duration: 67.8ms | Allocations: 6343)
rails-docker-web-1  |   Rendered layout layouts/application.html.erb (Duration: 376.3ms | Allocations: 39403)
rails-docker-web-1  | Completed 200 OK in 422ms (Views: 384.4ms | ActiveRecord: 6.6ms | Allocations: 43987)

http://localhost:3000/ にブラウザでアクセスしてスクリーンショットを取得して以下のエクセルにキャプチャを添付してるので、そちらからご確認頂けますようお願い致します。
ファイル名：ブラウザでアクセスしたスクリーンショット.xlsx

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+volumes:
+  db-data:
+
+services:
+  web:
+    build: .
+    command: >
+      bash -c "rails db:create &&
+              rails db:migrate &&
+              rails s -b 0.0.0.0"
```
Comment (yamaryo416) [inline docker-compose.yml:11]: コンテナを立ち上げるたびに`rails db:create`が走ってしまうので、`README.md`に一回のみ実行する旨を記述しましょう。

Comment (yuta-yamaura) [reply docker-compose.yml:11]: ご確認頂きありがとうございます。
上記記載したのでご確認をお願い致します。

Comment (yamaryo416) [reply docker-compose.yml:11]: こちらのファイルに、`rails db:create`は削除せず、
READEME.mdにて、docker-compose run --rm web rails. db:createを行う旨を記述しましょう。

**Review by yamaryo416 — APPROVED** [review]
LGTM!

**Pull request merged**